articles/tokens/index.md

```diff
@@ -49,7 +49,7 @@ The Refresh token is a long-lived token that is used to obtain a new [`id_token`
 
 ## Delegation Tokens
 
-The Auth0 [`id_token`](#auth0-idtoken-jwt)'s can be exchanged for another token, called a Delegation Token, that can be used to call either other application APIs registered as clients in the same Auth0 tenant or APIs represented by some types of application Addons registered in the same Auth0 tenant.
+The Auth0 [`id_token`](#id-token)'s can be exchanged for another token, called a Delegation Token, that can be used to call either other application APIs registered as clients in the same Auth0 tenant or APIs represented by some types of application Addons registered in the same Auth0 tenant.
```

Comment:
Should we remove delegation token docs or just remove the reference that you can use them to call other APIs? I know that Addons will be replaced by APIs (I assume at that point the delegation tokens will become obsolete) but I'm not clear on when this is going to happen.